ocs_ci/deployment/hub_spoke.py

```
@@ -5192,59 +5263,97 @@ def _cleanup_stale_oidc_providers(self):
+                            Bucket=self.oidc_bucket_name,
+                            Delete={"Objects": objects},
+                        )
+                self.s3_client.delete_bucket(Bucket=self.oidc_bucket_name)
+                logger.info(
+                    f"Deleted per-cluster OIDC bucket '{self.oidc_bucket_name}'"
+                )
+            except ClientError as e:
+                logger.error(
+                    f"Failed to delete per-cluster OIDC bucket "
+                    f"'{self.oidc_bucket_name}': {e}"
+                )
```

Ruff (0.16.1)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)